src/content/drafts/sourcecode-to-tokens-AST.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: How a parser translates text into tokens and then into an abstract syntax tree
+teaser: "You will learn how the source code in any proramming language is converted to token and then parsed to generate an AST. The AST can then be traversed to write and build custom rules in ESLint."
```

Comment:
```suggestion
teaser: "You will learn how the source code in any programming language is converted to a token and then parsed to generate an AST. The AST can then be traversed to write and build custom rules in ESLint."
```

Reply:
Since the source code is converted to multiple tokens, a token may not be applicable.
I will make the change suitably.

Reply:
sure.